Lab1b/lexic.txt

```diff
@@ -0,0 +1,37 @@
+Lab 1b - Lexic
+Alphabet:
+a.	Upper (A-Z) and lower case letters (a-z) of the English alphabet
+b.	Underline character '_'
+c.	Decimal digits (0-9)
+Lexic:
+a.	Special symbols, representing:
+o	Operators
+-	Arithmetic + - * / %
+-	Relational == < <= >= > <>
+-	Logical && || !
+-	Assignment =
+o	Separators : ; space [] {}
+o	Reserved words if else int char bool float for while do @
+b.	Identifiers (sequence of uppercase or lowercase letters and numbers, which starts with a letter)
+    IDENTIFIER = LETTER {LETTER | DIGIT}
+    LETTER = “A” | “B” | ... | “Z” | “a” | “b” | … | “z”
+    DIGIT = “0” | “1” | ... | “9”
+c.	Constants
+1.	Character
+CHARACTER = ‘LETTER’ | ‘DIGIT’
+
+2.	String
+STRING = “{LETTER | DIGIT}”
+
+3.	Integer - rule:
```

Comment:
Consider taking separatly the case when the number is 0.
Take into account the fact that a number that has minimum 2 digits can't start with 0.

Reply:
good point, thanks for pointing it out :D